Gemfile.lock

```diff
@@ -1,42 +1,55 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    addressable (2.3.7)
-    faraday (0.9.1)
+    addressable (2.3.8)
+    domain_name (0.5.20170404)
+      unf (>= 0.0.5, < 1.0.0)
+    faraday (0.9.2)
       multipart-post (>= 1.2, < 3)
-    jgrep (1.4.1)
-      json
-    json (1.8.3)
+    http-cookie (1.0.3)
+      domain_name (~> 0.5)
+    jgrep (1.5.0)
+    json (1.8.6)
+    mime-types (2.99.3)
+    minitest (5.10.3)
     multipart-post (2.0.0)
-    netrc (0.7.7)
-    octokit (3.7.1)
+    netrc (0.11.0)
+    octokit (3.8.0)
       sawyer (~> 0.6.0, >= 0.5.3)
-    rack (1.5.2)
+    rack (1.6.8)
     rack-protection (1.5.3)
       rack
-    rest_client (1.7.3)
-      netrc (~> 0.7.7)
+    rake (12.2.1)
+    rest-client (1.8.0)
+      http-cookie (>= 1.0.2, < 2.0)
```

Comment:
1.7 -> 1.8 in rest client is a bit of a risk? IIRC rest client can be a total pain, is this needed *before* we can add tests to verify it works?

Reply:
I didn't carefully check and just wanted to pin older than 2.0. Changing to 1.7.x

Reply:
Admittedly this repo is not live, we could merge it and no push to Openshift ...

Reply:
Let's keep this in shape to deploy.